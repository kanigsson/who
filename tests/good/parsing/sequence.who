type unit
logic := : int -> int -> unit
let f = (fun (x : int) -> {} x:= 0; 5 {})
