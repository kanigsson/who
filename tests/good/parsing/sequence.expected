type unit
logic :=  : int -> int -> unit
let
  f  = (fun (x:int) -> {(fun (cur:<{ }>) -> True)}
       let anon = (x := 0 : unit) in 5
       {(fun (old:<{ }>) -> (fun (cur1:<{ }>) -> (fun (anon1:int) -> True)))}) 
